installbuilder/datafiles/Base_DSC.data

```diff
@@ -169,14 +169,16 @@ EOF
 
 chmod u+x $tmpfile
 $tmpfile 1> /dev/null 2> /dev/null
-if [ $? -ne 0 ]; then
+tmpfile_ctypes=$?
+rm $tmpfile
+
+python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
Yes, I have verified through testing a while ago that, if a system does not have python-ctypes installed, the command line form will fail, and cause the error message to be printed out.

I like the idea of printing out a message if the commandline succeeds but the script form try fails, saying that it appears that /tmp is mounted as noexec and this may affect other DSC functionality.

As far as I know, the OMSAgent does not try to write to /tmp directly, but DSC may. If this is the case, then a message to this point would be useful.